README.md

```diff
@@ -191,19 +191,44 @@ CLOUDFLARE_APPLICATION_AUDIENCE=<CLOUFLARE_ACCESS_AUDIENCE_TAG>
 
 ### Docker Build and Run
 
-Build the Docker image
+Build the Docker image and run it using Docker Compose:
 
 ```
-docker build -t access .
+docker-compose up --build
```

Comment:
change this to `docker compose` since [`docker-compose` is deprecated](https://docs.docker.com/compose/migrate/#docker-compose-vs-docker-compose)

Reply:
addressing this